### DISCUSSION TITLE
Moving Quarkus to a Foundation

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#41109] [Quarkus Blog/Website]
# Moving Quarkus to a Foundation

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-in-a-foundation/

**Comment by smsajid** [comment]
An welcome move. I did see this coming but no so soon.

**Comment by luizpais** [comment]
That's great news! Shows the commitment to open source and the community ;)

**Comment by fercomunello** [comment]
Sounds great! :D

**Comment by jolucode** [comment]
Hello friends maybe you can help me 

[image.png](/user_uploads/11321/uVzfb5Sa9a7oNW78sJrmeUWR/image.png)


I have this problem when I traid to consume private access but some suggestions is add the next:

[image.png](/user_uploads/11321/tKxxfFWepEFpTZB0BotiC-fg/image.png)


Please do you have anysuggestions?

JDK 21
